pcsx2/gui/Dialogs/AboutBoxDialog.cpp

```diff
@@ -37,69 +37,55 @@ Dialogs::AboutBoxDialog::AboutBoxDialog(wxWindow* parent)
 		wxDefaultPosition, wxDefaultSize
 		)
 {
-	// [TODO] : About box should be upgraded to use scrollable read-only text boxes.
-
-	wxString developsString = wxsFormat(
-		L"avih, Refraction, rama, pseudonym, gregory.hainaut, turtleli"
-		L"\n\n"
-		L"%s: \n"
-		L"Arcum42, Aumatt, drk||raziel, "
-		L"cottonvibes, gigaherz, saqib, "
-		L"\n"
-		L"Alexey Silinov, Florin, "
-		L"goldfinger, Linuzappz, loser, "
-		L"Nachbrenner, shadow, Zerofrog, tmkk, Jake.Stine"
-		L"\n\n"
-		L"%s:\n"
-		L"Falcon4ever, Bositman, Akasha",
-		_("Previous versions"), _("Webmasters"));
-
-	wxString contribsString = wxsFormat(
-		L"ssakash, FlatOut, CK1, lightningterror, atomic83github, Pandubz, Prafull, Blyss Sarania, Nobbs66, Alessandro Vetere"
-		L"\n\n"
-		L"%s: \n"
-		L"ChickenLiver (Lilypad), Gabest (GSdx, Cdvdolio, Xpad), Ckemu, "
-		L"General Plot, KrossX, Devina, micove, black_wd, Belmont, "
-		L"\n"
-		L"BGome, _Demo_, Dreamtime, Hiryu and Sjeep, nneeve, F|RES, "
-		L"Shadow Lady, MrBrown, razorblade, Seta-san, Skarmeth, feal87, Athos"
-		L"\n",
-		_("Previous versions"));
-
-	wxFlexGridSizer& boxesContainer = *new wxFlexGridSizer(2, 0, StdPadding);
-	boxesContainer.AddGrowableCol(0, 1);
-	boxesContainer.AddGrowableCol(1, 1);
-
-	wxStaticBoxSizer& developsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
-	wxStaticBoxSizer& contribsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
-
-	pxStaticText& developsText = Text(developsString).SetMinWidth(350);
-	pxStaticText& contribsText = Text(contribsString).SetMinWidth(350);
-
-	developsBox += Heading(_("Developers")).Bold() | StdExpand();
-	developsBox += developsText | StdExpand();
-	contribsBox += Heading(_("Contributors")).Bold() | StdExpand();
-	contribsBox += contribsText | StdExpand();
-
-	boxesContainer += developsBox | StdExpand();
-	boxesContainer += contribsBox | StdExpand();
-
 	// Main layout
+
+	SetMinWidth(460);
+
 	*this += m_bitmap_logo | StdCenter();
 
-	*this += Text(_("PlayStation 2 Emulator"));
+	*this += Text(_("PlayStation 2 Emulator:"));
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Main Website"), L"https://pcsx2.net"
+	) | pxProportion(1).Center().Border(wxALL, 3);
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Support Forums"), L"https://forums.pcsx2.net"
+	) | pxProportion(1).Center().Border(wxALL, 3);
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("GitHub Repository"), L"https://github.com/PCSX2/pcsx2"
+	) | pxProportion(1).Center().Border(wxALL, 3);
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Contributors"), L"https://www.openhub.net/p/Pcsx2/contributors/summary"
```

Comment:
I wouldn't include this because it only really values coding contributors when there's other forms of contributors. But that's just my opinion.

Reply:
We do have a thank you for every contributor here https://github.com/PCSX2/pcsx2/pull/3059/files#diff-7242f63bb6c1fff6ed237d24f219c4e8R86

I vote to keep some sort of link, don't know about others.